From ca50a1d8388da5f2a482f6291ab2ff17d3efc52e Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Thu, 13 May 2010 19:25:36 -0400
Subject: [PATCH 050/391] lttng-instrumentation/revert-marker-remove2

revert marker remove2

LTTng tree (internal).
revert commit 115e8a288252ef748f34f8b7c1115c563d702eda

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 kernel/module.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/kernel/module.c b/kernel/module.c
index 020e3b9..3e51758 100644
--- a/kernel/module.c
+++ b/kernel/module.c
@@ -3048,6 +3048,7 @@ void module_layout(struct module *mod,
 		   struct modversion_info *ver,
 		   struct kernel_param *kp,
 		   struct kernel_symbol *ks,
+		   struct marker *marker,
 		   struct tracepoint *tp)
 {
 }
-- 
1.6.5.2

